### PR TITLE
Add support for bulk set route_entry api

### PR DESCRIPTION
**Pull request by kcudnik**

### CODE REVIEW

**Comment by lguohan** [comment]
test this please

**Comment by kcudnik** [comment]
yes, already tested

**Comment by lguohan** [comment]
I installed the redis and start the server.

https://sonic-jenkins.westus.cloudapp.azure.com/job/broadcom/job/sonic-sairedis-build-pr/76/consoleFull

**Comment by lguohan** [comment]
test this please

**Comment by kcudnik** [comment]
test this please

**Comment by lguohan** [comment]
test this please

**Comment by kcudnik** [comment]
hmm, still can't connect to socket

File: lib/inc/sairedis.h [hunk 1]
```diff
@@ -70,4 +70,137 @@ typedef enum _sai_redis_switch_attr_t
 
 } sai_redis_switch_attr_t;
 
+/*
+ * Those bulk APIs are provided as static functions since current SAI 0.9.4
+ * doesn't support bulk API.  Also note that SDK don't support bulk route API
+ * yet, so those APIs are introduced here to reduce number of calls to redis,
+ * in syncd bulk API will be splitted to separate call's until proper SDK
+ * support will be added.
+ *
```
Comment (rodnymolina) [inline lib/inc/sairedis.h]: Nice to have this feature at redis-db level. But what about SDK level? Are you aware of any upcoming SAI proposal to address bulk requests?

Comment (kcudnik) [inline lib/inc/sairedis.h]: Yes, i added pull request to support bulk API on sai level, please visit sai repo

Comment (kcudnik) [inline lib/inc/sairedis.h]: https://github.com/opencomputeproject/SAI/pull/516

**Comment by kcudnik** [comment]
unix socket permissions should be 777 for redis

File: lib/inc/sairedis.h (hunk 1 above)
Comment (rodnymolina) [inline lib/inc/sairedis.h]: thanks for the link!

**Comment by kcudnik** [comment]
test this please

**Pull request merged**